code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -559,3 +559,16 @@
 	flagpatch_national["flagpatch, nralakk"] = /obj/item/clothing/accessory/flagpatch/nralakk
 	flagpatch_national["flagpatch, hegemony"] = /obj/item/clothing/accessory/flagpatch/hegemony
 	gear_tweaks += new /datum/gear_tweak/path(flagpatch_national)
+
+/datum/gear/accessory/aodai
```

Comment:
Are the ao dai and non la intentionally not culture restricted?

Reply:
The non la wasn't previously, and reasonably an Earth character could use either. Should I restrict to both there and NHP?

Reply:
Sure.